internal/eva/cookbook.go

```diff
@@ -157,7 +154,7 @@ func (c *Client) DeleteAccountingRecipe(ctx context.Context, req DeleteAccountin
 		return nil, errors.New(fmt.Sprintf("Request failed with error: %s", resp.String()))
 	}
 
-	var jsonResp DeleteAccountingRecipeResponse
+	var jsonResp EmptyResponse
 	if err := json.Unmarshal([]byte(resp.Body()), &jsonResp); err != nil {
```

Comment:
`nitpick (non-blocking)`: If we just care to the status code, maybe we can skip "unmarshalling" of the json.

Reply:
The whole point of this is to in a later point have function that takes a generic request and provides a generic response type so that we can have a single method do the exact same request/response handling. But that comes at a later time. Would essentially replace 70% of our copy/pasted code.